src/main/java/appeng/block/networking/BlockCableBus.java

```diff
@@ -82,13 +82,19 @@
 {
 
 	private static final ICableBusContainer NULL_CABLE_BUS = new NullCableBusContainer();
-	public static Class<? extends TileEntity> noTesrTile;
-	public static Class<? extends TileEntity> tesrTile;
+	private static Class<? extends TileEntity> noTesrTile;
+	private static Class<? extends TileEntity> tesrTile;
+
 	/**
 	 * Immibis MB Support.
+	 *
+	 * It will look for a field named ImmibisMicroblocks_TransformableBlockMarker or
+	 * ImmibisMicroblocks_TransformableTileEntityMarker, modifiers, type, etc can be ignored.
 	 */
-	boolean ImmibisMicroblocks_TransformableBlockMarker = true;
-	int myColorMultiplier = 0xffffff;
+	@SuppressWarnings( "unused" )
+	private final static boolean ImmibisMicroblocks_TransformableBlockMarker = true;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00115)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Reorder the modifiers to comply with the Java Language Specification. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AModifiersOrderCheck)